practico_08/autogestion_alumnos/templates/dashboard.html

```diff
@@ -1,154 +1,103 @@
-{% extends 'base.html' %}
-
-{% block title %}
-    {{ super() }}
-    Home
-{% endblock %}
-
-{% block content %}
-<main>
+<!DOCTYPE html>
```

Comment:
I don't know where to write this so I'm writing it here, PLEASE PLEASE, WRITE RIGHT NAMES, images/1.png is not a good name, please, change all those image names, to something like logo-calendar or similar. Thanks!